apis/v1alpha2/validation/httproute.go

```diff
@@ -93,3 +99,36 @@ func validateHTTPBackendUniqueFilters(ref []gatewayv1a2.HTTPBackendRef, path *fi
 	}
 	return errs
 }
+
+// webhook validation of HTTPPathMatch
+func validateHTTPPathMatch(path *gatewayv1a2.HTTPPathMatch, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if path.Type == nil {
+		return append(allErrs, field.Required(fldPath.Child("pathType"), "pathType must be specified"))
+	}
+
+	switch *path.Type {
+	case gatewayv1a2.PathMatchExact, gatewayv1a2.PathMatchPrefix:
+		if !strings.HasPrefix(*path.Value, "/") {
```

Comment:
Although defaulting should generally remove the possibility of a nil value here, I think it's still best to add a nil check before trusting there is a value.

Reply:
https://github.com/kubernetes-sigs/gateway-api/pull/894/commits/a87064c9ac5a3d05ce51af84ae7083ccbf5ed3da